src/core/logging_manager.py

```diff
@@ -0,0 +1,129 @@
+        sys.stderr.flush()
+
+
+class CustomJsonFormatter(logging.Formatter):
+    """Custom JSON formatter."""
+
+    def format(self, record: logging.LogRecord):
+        """Custom json format for log message.
+
+        :param record:
+        :return:
+        """
+        log_message = {
+            "datetime": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
+            "name": record.name,
+            "level": record.levelname,
+            "filename": f"{record.filename}:{record.lineno}",
+            "message": record.getMessage(),
+        }
+        return json.dumps(log_message)
+
+
+class LoggingManager:
+    """Custom Loging Manager."""
+
+    _LOG_FILE_NAME = "appointment-scheduler.log"
+
+    @staticmethod
+    def init_logger_json_format():
+        """Setup logger in custom JSON format.
+
+        :return:
+        """
+        logger = logging.getLogger(__name__)
+        logger.setLevel(logging.DEBUG)
+
+        log_format = "%(log_color)s%(asctime)s - %(name)s - [%(levelname)s] -  %(filename)s:%(lineno)d - %(message)s"
+        color_formatter = colorlog.ColoredFormatter(
+            log_format,
+            log_colors={"DEBUG": "cyan", "INFO": "green", "WARNING": "yellow", "ERROR": "red", "CRITICAL": "bold_red"},
+            datefmt="%Y-%m-%d %H:%M:%S",
+        )
+
+        # File handler
+        if not os.path.exists(CommonPaths.log_path):
+            os.makedirs(CommonPaths.log_path)
+        file_handler = logging.FileHandler(
+            CommonPaths.log_path.joinpath(LoggingManager._LOG_FILE_NAME), encoding="utf-8"
+        )
+        console_handler = PrintHandler()
+
+        handlers = [file_handler, console_handler]
+        logging.basicConfig(level=logging.INFO, handlers=handlers, encoding="utf-8", force=True)
+        for handler in logging.root.handlers:
+            handler.setFormatter(color_formatter)
+            handler.setFormatter(CustomJsonFormatter())
+
+    @staticmethod
+    def setup_logger_string_format():
+        """Setup logger in custom string format.
+
+        :return:
+        """
+        root_logger = logging.getLogger(__name__)
+        root_logger.setLevel(logging.DEBUG)
+
+        log_format = "%(log_color)s%(asctime)s - %(name)s - [%(levelname)s] -  %(filename)s:%(lineno)d - %(message)s"
+        formatter = colorlog.ColoredFormatter(
+            log_format,
+            log_colors={"DEBUG": "cyan", "INFO": "green", "WARNING": "yellow", "ERROR": "red", "CRITICAL": "bold_red"},
+            datefmt="%Y-%m-%d %H:%M:%S",
+        )
+        if not os.path.exists(CommonPaths.log_path):
+            os.makedirs(CommonPaths.log_path)
+
+        file_handler = logging.FileHandler(
+            CommonPaths.log_path.joinpath(LoggingManager._LOG_FILE_NAME), encoding="utf-8"
+        )
+        console_handler = PrintHandler()
+        handlers = [file_handler, console_handler]
+        logging.basicConfig(level=logging.INFO, handlers=handlers, encoding="utf-8", force=True)
```

Comment:
@vasylboliuk basic config is supposed to be used only once in your code, you don't want to redefine it.